tests/cipethernetip/cip_ethernet_ip_simulator.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```

Comment:
There should be a unit test case as well.

Reply:
I don't understand how this has to be done.